src/Analyser/Database/DatabaseAnalyser.php

```diff
@@ -70,7 +70,7 @@ protected function analyseColumns(Table $table): void
 			$column->setNullable($col['nullable']);
 			$column->setType(Helpers::columnType($col['nativetype']));
 			$column->setDefault($col['default']);
-			$column->setOnUpdate(Strings::contains($col['vendor']['Extra'], 'on update'));
+			$column->setOnUpdate(Strings::contains($col['vendor']['extra'], 'on update'));
```

Comment:
Capitalized key was introduced by #16, someone should check proper behavior before any changes of that https://github.com/contributte/nextras-orm-generator/commit/b0002a1da7578586a83576d341e224567eaf79be